### DISCUSSION TITLE
Sticky hamburger menu

### DISCUSSION THREAD

**Discussion by wezykowski** [templaza/astroid-framework#579] [Q&A]
Hello!
I have an astroid template one. The menu is placed in the left position. The website temporarily operates at http://nauka.pzhkipe.org/
How to stick a hamburger on mobile devices at the top of the page?
Regards Jacky

**Comment by paurray** [comment]
@wezykowski 
It looks as if you figured this out or am I missing something?
Is this is your workflow?
https://docs.astroidframe.work/header/off-canvas-menu

<img width="360" alt="MenuOnMobileDevices" src="https://github.com/templaza/astroid-framework/assets/9163424/ad7ab19c-6b35-4dfc-b0ce-1ac75001d00d">
